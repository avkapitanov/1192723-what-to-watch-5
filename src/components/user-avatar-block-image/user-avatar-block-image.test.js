import React from "react";
import renderer from "react-test-renderer";
import UserAvatarBlockImage from "./user-avatar-block-image";
import {Router as BrowserRouter} from "react-router-dom";
import browserHistory from "../../browser-history";

const avatarUrl = `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/9.jpg`;

it(`UserAvatarBlockImage component render correctly`, () => {
  const tree = renderer.create(
      <BrowserRouter history={browserHistory}>
        <UserAvatarBlockImage
          avatarUrl={avatarUrl}
        />
      </BrowserRouter>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

