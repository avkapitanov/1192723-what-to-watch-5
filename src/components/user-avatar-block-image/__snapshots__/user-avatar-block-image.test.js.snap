// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserAvatarBlockImage component render correctly 1`] = `
<div
  className="user-block__avatar"
>
  <img
    alt="User avatar"
    height="63"
    onClick={[Function]}
    src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/9.jpg"
    width="63"
  />
</div>
`;
